src/color/setting.js

```diff
@@ -787,8 +1277,10 @@ p5.prototype.noFill = function() {
 };
 
 /**
- * Disables drawing the stroke (outline). If both
- * <a href="#/p5/noStroke">noStroke()</a> and
+ * Disables drawing points, lines, and the outlines of shapes.
```

Comment:
How about a simplified version that's similar to the previous version, like 'Disables drawing the outlines of shapes"? 
It seems like 'lines' might be a repeat of 'outline', and I'm not sure if we need mention 'points' here. 
What do you think? @limzykenneth and @davepagurek

Reply:
I think the nice part about having lines and points is that it's maybe not immediately clear that those would be drawn with the stroke color rather than the fill color, since those aren't outlines of shapes. So this is maybe a helpful extra detail to include

Reply:
Agreed with @davepagurek that the details would be helpful here.